ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-presubmits.yaml

```diff
@@ -308,6 +308,197 @@ presubmits:
             --template=/usr/local/e2e-gcp-crio \
             --target=e2e-gcp-crio
 
+  - name: pull-ci-openshift-openshift-ansible-e2e-gcp-upgrade-310-to-310x
+    agent: kubernetes
+    context: ci/prow/gcp-minor-upgrade
+    branches:
+    - release-3.10
+    rerun_command: "/test gcp-minor-upgrade"
+    always_run: true
+    trigger: "((?m)^/test( all| gcp-minor-upgrade),?(\\s+|$))"
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      volumes:
+      - name: job-definition
+        configMap:
+          name: prow-job-cluster-launch-e2e-upgrade
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp
+      containers:
+      - name: test
+        image: ci-operator:latest
+        resources: {"requests": {"cpu": "10m"},"limits":{"cpu":"500m"}}
+        volumeMounts:
+        - name: job-definition
+          mountPath: /usr/local/e2e-gcp
+          subPath: cluster-launch-e2e-upgrade.yaml
+        - name: cluster-profile
+          mountPath: /usr/local/e2e-gcp-cluster-profile
+        env:
+        - name: TEST_FOCUS
+          value: Suite:openshift/conformance/parallel
+        - name: JOB_NAME_SAFE
+          value: e2e-gcp
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: TEST_UPGRADE
+          value: "v3_10"
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              name: ci-operator-openshift-openshift-ansible
+              key: release-3.10.json
+        - name: RPM_REPO_BASEURL_REF
```

Comment:
Why is `$RPM_REPO` not valid? You should not need to set this.

Reply:
`.latest-rpms` doesn't give the repo directly, but rather a link to the latest repo for this branch. So later on it has to be fetched with `curl -q`